src/view/dataElements/xdmObject.jsx

```diff
@@ -23,53 +25,85 @@ import NodeEdit from "./xdmObject/components/nodeEdit";
 import validate from "./xdmObject/helpers/validate";
 import render from "../render";
 import fetchSchema from "./xdmObject/helpers/fetchSchema";
+import fetchSchemaList from "./xdmObject/helpers/fetchSchemaList";
+import findSchemaMetaById from "./xdmObject/helpers/findSchemaMetaById";
 import "./xdmObject.styl";
 
-const getInitialValues = ({ settings, schema }) => {
-  const value = (settings && settings.data) || {};
-  return getInitialFormState({ schema, value });
-};
-
-const XdmObject = ({ initInfo, formikProps, schema }) => {
+const XdmObject = ({
+  initInfo,
+  formikProps,
+  schemasMeta,
+  selectedSchemaMeta,
+  setSelectedSchemaMeta
+}) => {
   const { values: formState } = formikProps;
   const [selectedNodeId, setSelectedNodeId] = useState();
 
-  if (!schema) {
+  if (!schemasMeta.length) {
     return (
       <div className="u-flex u-fullHeight u-alignItemsCenter u-justifyContentCenter">
         <Alert variant="error" header="No Schema Found">
-          No configured XDM schema was found. Please ensure you have created an
-          edge configuration with a production environment.
+          No XDM schema was found. Please ensure you have created an XDM schema
+          within Adobe Experience Platform.
         </Alert>
       </div>
     );
   }
 
+  const schemaOptions = schemasMeta.map(schemaMeta => {
+    return {
+      value: schemaMeta.$id,
+      label: schemaMeta.title
+    };
+  });
+
   return (
-    <div className="u-flex u-fullHeight">
-      <div className="XdmObject-treeContainer u-flexShrink0 u-fullHeight u-overflowXAuto u-overflowYAuto">
-        <XdmTree
-          formikProps={formikProps}
-          selectedNodeId={selectedNodeId}
-          onSelect={setSelectedNodeId}
+    <div className="u-flex u-flexColumn u-fullHeight">
+      <FieldLabel
+        className="u-flex"
+        labelFor="schemaField"
+        label="Schema"
+        position="left"
+      >
+        <Select
+          id="schemaField"
+          data-test-id="schemaField"
+          className="u-widthAuto u-gapBottom"
+          options={schemaOptions}
+          value={selectedSchemaMeta.$id}
+          onChange={schemaMetaId => {
+            setSelectedNodeId(undefined);
+            setSelectedSchemaMeta({
+              $id: schemaMetaId
+            });
```

Comment:
Although you can get away with this currently, it introduces inconsistency in what is considered a "schema meta". Is it an object with `$id`? Is it an object with both `$id` and `title`? To be consistent and predictable, I recommend looking up the actual schema meta object using the ID before setting state:

```
const matchingSchemaMeta = schemaMetas.find(schemaMeta => schemaMeta.$id === schemaMetaId);
setSelectedSchemaMeta(matchingSchemaMeta);
```